packages/elements-core/src/utils/ref-resolving/resolvedObject.test.ts

```diff
@@ -236,4 +236,53 @@ describe('createResolvedObject', () => {
 
     expect(resolvedObject).toEqual(originalObject);
   });
+
+  it('removes object if contains an error for oneOf', () => {
+    const originalObject = {
```

Comment:
you can define originalObject and filteredObject at top. So, it won't be repeating in the test suit